releasenotes/notes/remove-deprecate-calsses-methods-7bd69606cc4ad61f.yaml

```diff
@@ -0,0 +1,5 @@
+---
+upgrade:
+  - |
+    Remove deprecated :class:`operators.Pauli` and :class:`pauli_group`.
+    Remove deprecated methods in :class:`Pauli`
```

Comment:
Please can this be updated to be clearer: `Pauli` isn't removed, it's just a particular import path of it that's to be deprecated.  `pauli_group` is removed.

We should also list the deprecated functionality in `Pauli`.  If there are any public methods that have been removed, we should list those.  I think a lot of them were private methods, though, which were accessed via particular paths through the initialiser - we should list the input forms that are no longer possible, and why they've been removed / what the alternatives are.

Reply:
Done in d60935360.